src/libs/ReportUtils.ts

```diff
@@ -8407,6 +8407,52 @@ function isExpenseReportWithoutParentAccess(report: OnyxEntry<Report>) {
     return isExpenseReport(report) && report?.hasParentAccess === false;
 }
 
+function isExpensifyAndCustomerChat(participantsContext: OnyxEntry<PersonalDetailsList>, report: OnyxInputOrEntry<Report>): boolean {
```

Comment:
```suggestion
function isExpensifyAndCustomerChat(participantsContext: OnyxEntry<PersonalDetailsList>, report: OnyxInputOrEntry<Report>): boolean {
```

I don't think this is a correct name for this function. We're also checking if the chat is policyExpenseChat and returning true, and in those cases, it's not a `expensify and customer` chat. We should probably rename this to something like `shouldUnmaskChat` or something like that.

Reply:
Thank you.
Done changed @danieldoglas